sample-apps/discount-functions-sample-app/extensions/product-discount/src/main.rs

```diff
@@ -109,21 +64,38 @@ fn build_result(value: f64, targets: Vec<Target>) -> FunctionResult {
 mod tests {
     use super::*;
 
-    fn input(configuration: Option<Configuration>) -> input::Input {
+    fn input(configuration: Option<input::Configuration>) -> input::Input {
         let input = r#"
         {
-            "merchandiseLines": [
-                { "variant": { "id": "gid://shopify/ProductVariant/0" } },
-                { "variant": { "id": "gid://shopify/ProductVariant/1" } }
-            ]
+            "cart": {
+                "lines": [
+                    {
+                        "id": "gid://shopify/CartLine/0",
+                        "merchandise": {
+                            "id": "gid://shopify/ProductVariant/0"
+                        }
+                    },
+                    {
+                        "id": "gid://shopify/CartLine/1",
+                        "merchandise": {
+                            "id": "gid://shopify/ProductVariant/1"
+                        }
+                    }
+                ]
+            },
+            "discountNode": {
+                "metafield": {
+                    "value": "{}"
+                }
+            }
         }
         "#;
```

Comment:
Nit: I think it would be preferable to construct a JSON value using [`json!`](https://docs.serde.rs/serde_json/macro.json.html) and then [`serde_json::from_value`](https://docs.serde.rs/serde_json/fn.from_value.html) than construct an actual JSON string like right here